src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -0,0 +1,17 @@
+package org.broadinstitute.dsde.workbench.leonardo.service
+
+import akka.actor.ActorSystem
+import akka.stream.Materializer
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.workbench.leonardo.dao.{DataprocDAO, GoogleDataprocDAO}
+import org.broadinstitute.dsde.workbench.leonardo.model.{ClusterRequest, ClusterResponse}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class LeonardoService(gdDAO: DataprocDAO)(implicit val system: ActorSystem, val materializer: Materializer, val executionContext: ExecutionContext) extends LazyLogging {
```

Comment:
We can remove them from several places.  I made a commit that you can cherry-pick here if you like: https://github.com/broadinstitute/leonardo/commit/05692a1f1812a09aa6b7cc39f3de36291cb35780